tests/Functional/Doctrine/Phpcr/RouteProviderTest.php

```diff
@@ -106,9 +116,11 @@ public function testGetRouteCollectionForRequestNonPhpcrUrl()
         $collection = $this->repository->getRouteCollectionForRequest(Request::create('http:///'));
         $this->assertInstanceOf(RouteCollection::class, $collection);
         $this->assertCount(1, $collection);
-        $routes = $collection->all();
-        list($key, $route) = each($routes);
-        $this->assertEquals(self::ROUTE_ROOT, $key);
+
+        $iterator = $routes->getIterator();
```

Comment:
here we called it $collection, so this fails.

best rename $collection above to $routes so that we are consistent.